src/Provider/Mollie.php

```diff
@@ -72,6 +72,42 @@ public function __construct(array $options = [], array $collaborators = [])
 		}
 	}
 
+	/**
+	 * @var string
+	 */
+	private $mollieApiUrl = self::MOLLIE_API_URL;
+
+	/**
+	 * @var string
+	 */
+	private $mollieWebUrl = self::MOLLIE_WEB_URL;
+
+	/**
+	 * Define Mollie api URL
+	 *
+	 * @param string $url
+	 * @return Mollie
+	 */
+	public function setMollieApiUrl ($url)
```

Comment:
**NBC**: Setting this value in the constructor with checking the `$options` parameter makes more sense to me. Same for `setMollieWebUrl()`.